src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
+@import 'variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  padding: $calendar-padding;
+  width: calc((7 * $day-size) + (6 * $day-gap) + (2 * $calendar-padding));
+  gap: $day-gap;
+}
+
+.calendar__day {
+  position: relative;
+  width: $day-size;
+  height: $day-size;
+  box-sizing: border-box;
```

Comment:
Since `box-sizing: border-box;` is recommended to be set globally, there is no need to repeat it for the `.calendar__day` class.